addons/udes_priorities/models/stock_picking_batch.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from odoo import api, fields, models
+
+
+class StockPickingBatch(models.Model):
+    _inherit = "stock.picking.batch"
+
+    priority = fields.Selection(selection="_get_priorities_for_selection")
```

Comment:
Name consistency with other models, in this case we have `_` is there a reason to have it in this one but not the others?